src/weather.cpp

```diff
@@ -61,44 +61,37 @@ static bool is_player_outside()
 /**
  * Glare.
  * Causes glare effect to player's eyes if they are not wearing applicable eye protection.
+ * @param intensity Level of sun brighthess (1 = clear, 2 = sunny)
```

Comment:
Seems like this should be an `enum`, as hinted by the explanation in the comment: there are only two possible values and you have given them names in the comment. Why not give those two values names in the code itself?

Reply:
Fixed, thanks